applications/DEMApplication/custom_elements/cluster3D.cpp

```diff
@@ -136,7 +136,7 @@ namespace Kratos {
         else if (contact_info_element) ElementNameString= "ContactInfoSphericParticle3D";
         else ElementNameString= "SphericParticle3D";
 
-        if (!continuum_strategy && breakable) KRATOS_THROW_ERROR(std::runtime_error,"Breakable cluster elements are being used inside a non-deformable strategy. The program will now stop.","")
+        KRATOS_ERROR_IF(!continuum_strategy && breakable) << "Breakable cluster elements are being used inside a non-deformable strategy. The program will now stop." << std::endl;
```

Comment:
Why is - KRATOS_ERROR_IF_NOT- not being used here? Whats the criteria between KRATOS IF and KRATOS IF NOT?

Reply:
Because inside the `if` there was two variables to be checked: one has `!` but the other one hasn't. I can use `KRATOS_ERROR_IF_NOT`, but then I need to change the `&&` to `||`. I thought it was better to leave it with `KRATOS_ERROR_IF`